gulpfile.babel.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
```

Comment:
💯 to this! Meant to suggest it earlier since I saw it in another project. Babelified gulpfiles are awesome for consistency.